css/pages/faq.css

```diff
@@ -25,75 +20,50 @@ h1 {
   margin-bottom: 1.5rem;
 }
 
-.questions-group {
-  margin-left: 10%;
-}
-
-.faq__list {
-  margin: 10px 0 0 0;
-  padding: 0;
-  list-style: none;
-}
-.faq__list li {
-  padding-left: 17px;
-  position: relative;
-  margin-bottom: 5px;
-}
-.faq__list li::before {
-  content: "";
-  background: url(../../img/logos/list-icon.png);
-  width: 15px;
-  height: 15px;
-  display: inline-block;
-  top: 2px;
-  position: absolute;
-  left: 0;
-  background-size: cover;
-}
-footer {
-  position: relative;
-  height: 50px;
-  bottom: 0px;
-  left: 0px;
-  right: 0px;
-  margin-bottom: 0px;
-}
-
-a {
-  text-decoration: none;
-  color: #0000ff;
-  font-weight: 400;
-}
-
 .faq {
   background: #fff;
   border-radius: 0.5rem;
   box-shadow: 0 5px 15px rgba(0, 0, 0, 0.1);
-  padding: 1rem 1rem 0 1rem;
+  padding: 1rem 1rem 1rem 1rem;
```

Comment:
Better to use just `padding: 1rem` 
`When one value is specified, it applies the same padding to all four sides.`
Source: https://developer.mozilla.org/en-US/docs/Web/CSS/padding

Reply:
Ya sure @swarajpure . 
Thankyou for suggesting.

Reply:
Change Done @swarajpure . May you Review it again.

Reply:
Since all four values are the same, use `padding: 1rem;` @swarajpure @gauravchauhan2403